src/components/cross-chain.js

```diff
@@ -128,14 +128,16 @@ class CrossChainRecoveryForm extends Component {
 
     for (const txid of txids) {
       try {
+        // Remove the xprv since the default is an empty string instead of undefined
+        const xprv = (passphrase) ? undefined : prv;
```

Comment:
It is not clear to me why a "truthy" passphrase should cause the `xprv` to be set to undefined. Is it possible that you mean to check that the user is only sending either a passphrase or an `xprv`? Please make your check more explicit.

Please be sure to include a bit more information in the commit message about why you're changing this.

Reply:
agree with Mark. Please add a comment about this!